owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/SettingsFragmentMoreSectionTest.kt

```diff
@@ -0,0 +1,380 @@
+        intending(hasAction(ACTION_SENDTO)).respondWith(intentResult)
+        intended(
+            allOf(
+                hasAction(ACTION_SENDTO), hasExtra(
+                    EXTRA_SUBJECT, String.format(
+                        context.getString(R.string.recommend_subject),
+                        context.getString(R.string.app_name)
+                    )
+                ),
+                hasExtra(
+                    EXTRA_TEXT,
+                    String.format(
+                        context.getString(R.string.recommend_text),
+                        context.getString(R.string.app_name),
+                        context.getString(R.string.url_app_download)
+                    )
+                ),
+                hasFlag(Intent.FLAG_ACTIVITY_NEW_TASK)
+            )
+        )
+    }
+
+    @Ignore
+    @Test
+    fun feedbackOpensSender() {
+        launchTest()
+
+        onView(withText(R.string.prefs_feedback)).perform(click())
+        val intentResult = Instrumentation.ActivityResult(Activity.RESULT_OK, Intent())
+        intending(hasAction(ACTION_SENDTO)).respondWith(intentResult)
+        intended(
+            allOf(
+                hasAction(ACTION_SENDTO),
+                hasExtra(
+                    EXTRA_SUBJECT,
+                    "Android v" + BuildConfig.VERSION_NAME + " - " + context.getText(R.string.prefs_feedback)
+                ),
+                hasData(Uri.parse(context.getString(R.string.mail_feedback))),
+                hasFlag(Intent.FLAG_ACTIVITY_NEW_TASK)
+            )
+        )
+    }
+
+    @Ignore
+    @Test
+    fun privacyPolicyOpensPrivacyPolicyActivity() {
+        launchTest()
+
+        onView(withText(R.string.prefs_privacy_policy)).perform(click())
+        intended(hasComponent(PrivacyPolicyActivity::class.java.name))
+    }
+
+    @Test
+    fun imprintOpensUrl() {
+        every { settingsViewModel.getImprintUrl() } returns "https://owncloud.com/mobile"
+
+        launchTest()
+
+        onView(withText(R.string.prefs_imprint)).perform(click())
+        val intentResult = Instrumentation.ActivityResult(Activity.RESULT_OK, Intent())
+        intending(hasAction(Intent.ACTION_VIEW)).respondWith(intentResult)
+        intended(hasData("https://owncloud.com/mobile"))
+    }
+
+    companion object {
+        private const val PREFERENCE_SECURITY_CATEGORY = "security_category"
+        private const val PREFERENCE_MORE_CATEGORY = "more_category"
+        private const val PREFERENCE_HELP = "help"
+        private const val PREFERENCE_SYNC_CALENDAR_CONTACTS = "syncCalendarContacts"
+        private const val PREFERENCE_RECOMMEND = "recommend"
+        private const val PREFERENCE_FEEDBACK = "feedback"
+        private const val PREFERENCE_PRIVACY_POLICY = "privacyPolicy"
+        private const val PREFERENCE_IMPRINT = "imprint"
+        private const val PREFERENCE_ABOUT_APP = "about_app"
+    }
+}
```

Comment:
Try to add an empty line at the end of files 👍

Reply:
My fault 😬